tests/unit/test_ip.py

```diff
@@ -433,3 +444,7 @@ def test_ipaddress_interface(data):
 @pytest.mark.parametrize("data", IP_NETWORK)
 def test_ipaddress_network(data):
     assert ip.ipaddress_network(**data["sent"]) == data["received"]
+
+@pytest.mark.parametrize("data", IS_CLASSFULL)
+def test_is_classful(data):
+    assert ip.is_classfull(**data["sent"]) == data["received"]
```

Comment:
```suggestion
    assert ip.is_classfull(**data["sent"]) == data["received"]

```

Reply:
I think this got fixed when I ran black, there is a blank line at the end of the file now.